rpc/documentation/public_endpoints/getnodeinfo.md

```diff
@@ -8,7 +8,7 @@ None
 
 |     Parameter    |     Type      |                  Description                  |
 |:----------------:|:-------------:|:---------------------------------------------:|
-| `is_bootnode`    | bool          | Flag indicating if the node is a bootnode     |
+| `node_type`      | NodeType      | Flag indicating if the node type              |
```

Comment:
only natural to include this change here too :+1:

Reply:
Nice type there too 😆